### PR TITLE
Cleanup to prepare v1.2

### PR DESCRIPTION
**Pull request by JostMigenda**
In preparation for releasing v1.2, this PR contains documentation fixes and removes deprecated parameters from sample notebooks/scripts.

Ideally, these sorts of things should be included in the same PR that introduces the changes; it looks like both PR authors and reviewers (myself included) missed that. Wonder if it’s worth adding [a PR template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository) with checkmarks for docs, tests, sample code, etc.?

### CODE REVIEW

File: python/snewpy/models/ccsn.py [hunk 1]
```diff
@@ -117,36 +112,36 @@ def __init__(self, filename):
...
 class Tamborra_2014(_GarchingArchiveModel):
-    """Set up a model based on 3D simulations from [Tamborra et al., PRD 90:045032, 2014](https://arxiv.org/abs/1406.0006). Data files are from the Garching Supernova Archive.
+    """Model based on 3D simulations from `Tamborra et al., PRD 90:045032, 2014 <https://arxiv.org/abs/1406.0006>`_.
+    Data files are from the `Garching Supernova Archive`_.
     """
     pass
 
 class Bollig_2016(_GarchingArchiveModel):
-    """Set up a model based on simulations from Bollig et al. (2016). Models were taken, with permission, from the Garching Supernova Archive.
+    """Model based on simulations from Bollig et al. (2016). Models were taken, with permission, from the `Garching Supernova Archive`_.
```
Comment (Sheshuk) [inline python/snewpy/models/ccsn.py:121]: Missing citation

File: python/snewpy/models/ccsn.py [hunk 2]
```diff
@@ -117,36 +112,36 @@ def __init__(self, filename):
...
+    """Model based on SASI-dominated simulations from `Walk et al.,
+    PRD 101:123013, 2019 <https://arxiv.org/abs/1910.12971>`_. Data files are
+    from the `Garching Supernova Archive`_.
     """
     pass
 
 
 class OConnor_2013(PinchedModel):
-    """Set up a model based on the black hole formation simulation in O'Connor & Ott (2013). 
+    """Model based on the black hole formation simulation in O'Connor & Ott (2013).
```
Comment (Sheshuk) [inline python/snewpy/models/ccsn.py:141]: Missing citation

File: python/snewpy/models/ccsn.py [hunk 3]
```diff
@@ -185,11 +180,10 @@ def __init__(self, base, mass=15, eos='LS220'):
 
 
 class OConnor_2015(PinchedModel):
-    """Set up a model based on the black hole formation simulation in O'Connor (2015). 
+    """Model based on the black hole formation simulation in O'Connor (2015).
```
Comment (Sheshuk) [inline python/snewpy/models/ccsn.py:183]: Missing citation

File: python/snewpy/models/ccsn.py [hunk 4]
```diff
@@ -226,11 +220,10 @@ def __init__(self, filename, eos='LS220'):
         super().__init__(simtab, metadata)
 
 class Zha_2021(PinchedModel):
-    """Set up a model based on the hadron-quark phse transition models from Zha et al. 2021. 
+    """Model based on the hadron-quark phse transition models from Zha et al. 2021.
```
Comment (Sheshuk) [inline python/snewpy/models/ccsn.py:223]: Missing citation

File: python/snewpy/models/ccsn.py [hunk 5]
```diff
@@ -327,11 +319,10 @@ def __init__(self, filename, eos='LS220'):
 
 
 class Kuroda_2020(PinchedModel):
-    """Set up a model based on simulations from Kuroda et al. (2020)."""
+    """Model based on simulations from Kuroda et al. (2020)."""
```
Comment (Sheshuk) [inline python/snewpy/models/ccsn.py:322]: Missing citation

**Review by Sheshuk — CHANGES_REQUESTED** [review]
The cleanup looks good. But we need proper citations for all the models, and probably a better description for the `ccsn` module.

**Comment by Sheshuk** [comment]
Although those missing citations seem to be more fitting to the #180

**Review by Sheshuk — APPROVED** [review]
Looks good!

**Comment by JostMigenda** [comment]
Thanks for the commits and comments!

**Pull request merged**